### DISCUSSION TITLE
Spawn another android app activity from inside app built with briefcase

### DISCUSSION THREAD

**Discussion by thumbarnirmal** [beeware/briefcase#632] [Q&A]
Hi there,

I was wondering if it is possible to spawn activity of other android app from inside an app that I am making using briefcase and toga library/framework. I am trying to open e.g. com.android.documentsui.files.FilesActivity from inside the app.

I am new to this library and this is my first project on it. Please let me know if there are docs around this that I can read. I couldn't find relevant information in briefcase docs, hence posting this question.

Thanks!

**Comment by freakboy3742** [comment]
Unfortunately, this isn't documented; but it is at least *possible*.

The Android app implementation contains an `intent_result` method; you can pass it any Android intent. You can see an example of how it is used in PR #1158.
